google/cloud/dataproc_spark_connect/session.py

```diff
@@ -494,15 +498,21 @@ def _wait_for_session_available(
             )
 
         def _display_session_link_on_creation(self, session_id):
-            session_url = f"https://console.cloud.google.com/dataproc/interactive/{self._region}/{session_id}?project={self._project_id}"
+            session_url = f"{_DATAPROC_SESSIONS_BASE_URL}/{self._region}/{session_id}?project={self._project_id}"
             plain_message = f"Creating Dataproc Session: {session_url}"
-            html_element = f"""
+            if environment.is_colab_enterprise():
```

Comment:
This actually does not distinguish between BQ Studio and Colab Enterprise - we need to have a link that works for both, in BQ Studio it should open side-panel and in Colab Enterprise new browser tab.